src/consts.ts

```diff
@@ -1,10 +1,12 @@
+export type NodeStatusValue = 'uninitialized' | 'loading' | 'done' | 'alert' | 'failed';
+export type NodeStatuses = Record<string, NodeStatusValue>;
```

Comment:
I think this type is not necessary. Also it is not used outside of this file, so no need for exporting.